ot/utils.py

```diff
@@ -156,8 +160,9 @@ def projection_sparse_simplex(V, max_nz, z=1, axis=None, nx=None):
     -------
```

Comment:
while you're at it you fix them int he 3 lines above.
<img width="430" height="109" alt="Screenshot 2025-07-25 at 17 41 13" src="https://github.com/user-attachments/assets/136ab225-c44c-4491-a7ba-ba0e6a933aee" />